### DISCUSSION TITLE
Why doesn't <Flex /> support space-around and space-evenly?

### DISCUSSION THREAD

**Discussion by andrecasal** [radix-ui/themes#96] [Q&A]
I'm intrigued by the fact that `<Flex />` doesn't support `space-around` and `space-evenly`.

Is this a conscious decision (i.e. "it's bad practice to use `space-around` and `space-evenly`, so we're not going to add it") or hasn't been implemented yet?

**Comment by andrecasal** [comment]
Also no `align-content` when `flex-wrap`'s enabled.

**Comment by vladmoroz** [comment]
Nothing bad about space-around and space-evenly, we didn't add them to the prop values to keep the ergonomics clean because they are rarely used and are less intuitive as a layout method. They are still available if you need them once in a blue moon via inline styles or other styling solutions.

Same about align-content.

**Reply by andrecasal** [reply]
Very interesting. I think it would be a great addition to the documentation to add that note.

**Reply by kgaregin** [reply]
@vladmoroz I understand Radix team does its best to make the dev and user experience as good as possible. I enjoy using Radix a lot and I started just recently. With all due respect, the rule of thumb in any design always was to not sacrifice core functionality in favor of "good look".

**Comment by Azoonex** [comment]
I think if it is added justify="around", it can make Flex more complete

**Comment by varun-d** [comment]
I am slightly confused too. I use space between for header menu items, left side logo and right side "stuff". Following the Auto Layout system in Figma.
